index.js

```diff
@@ -211,14 +210,14 @@ RedisClient.prototype.do_auth = function () {
     self.send_anyway = true;
     self.send_command("auth", [this.auth_pass], function (err, res) {
         if (err) {
-            if (err.toString().match("LOADING")) {
+            if (/LOADING/.test(err.toString())) {
```

Comment:
performance nit: creating these regular expressions as vars and moving them out of the send_command code-path would be a slight improvement.